.travis-ci.sh

```diff
@@ -75,6 +75,16 @@ function build_one {
     echo "End   availability check."
     case $TRAVIS_OS_NAME in
     linux)
+      # we need fresh gcc and binutils, maybe...
+      if [ `opam install --dry-run $pkg | grep -c mirage-entropy-xen` -gt 0 ] ; then
```

Comment:
Is this something we can fix by forking the PPA and correcting it?  Would be really nice to be able to provide reasonable installation instructions for Precise, although Trusty is the primary target.